BUILD.bazel

```diff
@@ -1,11 +1,28 @@
 load("@bazel_gazelle//:def.bzl", "gazelle", "gazelle_binary")
 load("@npm//:defs.bzl", "link_js_packages")
+load("@acorn__8.4.0__links//:link_js_package.bzl", link_acorn = "link_js_package")
+load("//js:defs.bzl", "link_js_package", "link_js_package_dep")
 
 # Link all packages from the /WORKSPACE translate_pnpm_lock in the package.json to
 # bazel-bin/examples/node_modules as well as the virtual store since this package.json
 # is the root of the pnpm workspace
 link_js_packages()
 
+# Link the acorn package, which was fetched separately with npm_import from /WORKSPACE, to the
+# virtual store in bazel-bin/node_modules
+link_acorn()
+
+# Linking a first party dependency to the virtual store in bazel-bin/node_modules; `direct` is set to
+# False so we don't link this first party dependency as a direct dependency here
```

Comment:
this comment wasn't enough for me to understand what direct=False is doing

Reply:
Updated to
```
    # `direct` set to False as an example of *not* also linking this first-party dependency as a
    # direct dependency in the package at bazel-bin/node_modules/@mycorp/mylib.
    direct = False,
```